layouts/partials/inline-spotlight/inline.css

```diff
@@ -419,4 +419,25 @@
 	'.tvp-product-info {'+
     'display: {{- ($.Scratch.Get "settings").product_item_info_display_mobile -}};' +
 	'}'+
+	'.tvp-slide-up {'+
+    'display: {{- ($.Scratch.Get "settings").product_item_show_hide_feature_mobile_display -}};' +
+	'}'+
+	'.tvp-slide-up > .tvp-minus {'+
```

Comment:
Please merge the rules to have one css declaration

Reply:
done